Applications/GesturePod/onComputer/README.md

```diff
@@ -0,0 +1,46 @@
+GesturePod *(Gesture Recognition Pipeline)*
+=========================
+
+## Table of Contents
+
+- [About](#about)
+- [Quick Start](#quick-start)
+- [Docs](#docs)
+- [Also](#also)
+
+## About
+
+GesturePod is a plug-and-play, gesture recognition device. 
+
+As the GesturePod involves hardware, this implements GesturePod's gesture recognition pipeline on your computer system. We have stored sensor data collected from the GesturePod. This pre-stored sensor data is used for gesture recognition. You can now play around with GesturePod on your computer!
+
+## Quick Start
+
+1. Installation requires 
+	- [git](https://git-scm.com/), 
+	- [make](https://www.gnu.org/software/make/)
+	- [g++](https://gcc.gnu.org/).
+
+2. Clone the repo and navigate to this directory
+	```
+	git clone https://github.com/Microsoft/EdgeML
+	cd EdgeML/Applications/GesturePod/onComputer
+	```
+3. Compile and build the code
+	```
+	make
+	```
+4. Run
+	```
+	./gesturepodsim
+	```
+5. By default the program runs for data file included in ```./data/```. This data file has the following gesture: *Double Taps*.
```

Comment:
By default, the program reads data from the `./data/FILENAME` file which contains space-separated IMU data collected during double taps.